src/ast/ddl.rs

```diff
@@ -23,15 +23,60 @@ use std::fmt;
 pub enum AlterTableOperation {
     /// `ADD <table_constraint>`
     AddConstraint(TableConstraint),
+    AddColumn {
+        has_column_identifier: bool,
```

Comment:
I think this represents whether the `COLUMN` keyword is there, and `ADD` vs `ADD COLUMN` is purely a stylistic difference? In that case I think we could drop this from the AST, parse `COLUMN` [like this](https://github.com/andygrove/sqlparser-rs/blob/fab6e2827195b6067cac5e5b3a1a9bb6bc161ae7/src/parser.rs#L1315), and prefer  `ADD COLUMN` when serializing back to string. This is mainly to avoid coming up with a less confusing name than `has_column_identifier`.